Tasks/DeleteFilesV1/deletefiles.ts

```diff
@@ -85,6 +87,22 @@ tl.setResourcePath(path.join(__dirname, 'task.json'));
         }
     }
 
+    // if there wasn't an error, check if there's anything in the folder tree other than folders
+    // if not, delete the root as well
+    if (removeSourceFolder && !errorHappened) {
+        foundPaths = tl.find(sourceFolder);
```

Comment:
Do we need to load this again?

Reply:
I wanted to grab the folder tree again, otherwise `foundPaths` will contain items that were (hopefully) deleted. In the success case, it should be an empty set.

Reply:
Makes sense, in that case I think we should ensure there is 1 folder only before doing anything.

Reply:
If you have time it would be good to add tests in here.

Reply:
I agree; I wish I had the time. If DeleteFiles already had a Tests folder ready to go, it wouldn't be so bad but I was having trouble getting the various packages to play nicely together. I'll try again next or next-next sprint.